addon/components/docs-header/search-results/index.hbs

```diff
@@ -1,6 +1,10 @@
-<div {{did-insert (perform this.search)}} {{did-update (perform this.search) @query}} ...attributes>
+<div
```

Comment:
Not exactly sure why my IDE wants to reformat everything in this project. Unless I'm missing some settings and actually my IDE is misconfigured, then might as well. Happy to revert if you think it makes too much noise.

Reply:
@MichalBryxi seems like you have prettier